guide/13-managing-arcgis-applications/hub-premium-guide.ipynb

```diff
@@ -0,0 +1,471 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "194ed88e",
+   "metadata": {},
+   "source": [
+    "# Introduction to ArcGIS Hub - Premium features\n",
+    "\n",
+    "ArcGIS Hub Premium enables organizations, like local non-profits, governments, or schools, to share a project or update with their community by starting an initiative or creating a site and organizing content and teams to share web maps, apps, datasets, and more with the public. Hub Premium also supports creating and sharing events related to your initiative using hub's built-in events management system. Click [here](https://www.esri.com/en-us/arcgis/products/arcgis-hub/resources) to learn more about ArcGIS Hub.\n",
+    "\n",
+    "In this notebook, we will explore the Hub Premium capabilities of __Initiatives__ and __Events__."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "e1bcd86c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from arcgis.gis import GIS"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3578e60b",
+   "metadata": {},
+   "source": [
+    "First, we will sign in to an organization with Hub Premium enabled to demonstrate these workflows. \n",
+    "\n",
+    "Note: In order to replicate these workflows for yourself, you may have to verify the features available to you, per your ArcGIS Hub licensing. \n",
+    "\n",
+    "You may also want to look at these examples for site and page creation and cloning using __ArcGIS Basic__ (insert live URL) and site and page layout and theme editing using __Enterprise Sites__ (insert live URL) to see the other ways of working with your Hub using the ArcGIS API for Python. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "c1423e98",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Enter password: ········\n"
+     ]
+    }
+   ],
+   "source": [
+    "gis = GIS(\"https://www.arcgis.com\", \"mmajumdar_dcdev\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "723532c9",
+   "metadata": {},
+   "source": [
+    "Next, we will connect to the Hub associated with this Web GIS."
```

Comment:
* it seems a better way to phrase this would be to avoid creating the impression that we are _connecting_ since we've already logged in:
  * _Next, we will initialize a [Hub](/python/api-reference/arcgis.apps.hub.html#hub) object to create link initiatives, sites and pages with this Web GIS._ or something along those lines?